### ISSUE TITLE
Question about pix2pix discriminator loss function

### ISSUE THREAD

**Issue by zhaozj89** [eriklindernoren/Keras-GAN#71]
I do not understand why discriminator loss function of pix2pix is written as `mse`. I think it should be something like:

    def mean_log_error(y_true, y_pred):
        return K.mean(K.log(K.square(y_pred - y_true)+1e-12), axis=-1)

But when I changed to the custom loss function `mean_log_error`, the training result looks not right.

What could be the problem? Thanks.
